### PR TITLE
IOP: affiliations cleaning

### PR DESCRIPTION
**Pull request by ErnestaP**

### CODE REVIEW

File: hepcrawl/extractors/jats.py [hunk 1]
```diff
@@ -106,7 +106,13 @@ def _get_authors(self, node):
             for aff in affiliations:
                 # checking is the aff. value captured by xpath is just new line
                 if self._clean_aff(aff).split():
-                    affiliations_values.append({'value': self._clean_aff(aff)})
+                    remove_doubled_commas = "".join(self._clean_aff(aff).split(" ,"))
+                    try:
+                        remove_doubled_commas = re.search('^(,\s)(.*)(,$)', remove_doubled_commas).group(2).strip()
```
Comment (drjova) [inline hepcrawl/extractors/jats.py:111]: @ErnestaP  it's weird, this regex doesn't work for me, could we please make a unit test?

Comment (ErnestaP) [reply hepcrawl/extractors/jats.py:111]: I should name the regex pattern to reflect what it does: 
'^(,\s)(.*)(,$)' pattern removes the commas in the end and beginning of the string. 
I had to keep the name of`remove_doubled_commas` to have the variable accessible after try/except block. I will improve this part of the code

File: hepcrawl/extractors/jats.py [hunk 2]
```diff
@@ -83,7 +83,7 @@ def _clean_aff(self, aff):
         for el in root:
             if el.tag == 'label':
                 root.remove(el)
-        return ', '.join(root.itertext())
+        return ''.join(root.itertext())
```
Comment (ErnestaP) [inline hepcrawl/extractors/jats.py:86]: this was adding additional commas, after every parser part. Let's say our XML looks like this:
 ```<aff id="AFF1">Institute of Science and Engineering<institution>Shimane University</institution>,Matsue 690-8504, <country country="JP">Japan</country></aff>```

All the string values: `["Institute of Science and Engineering", "Shimane University", ",Matsue 690-8504, ", "Japan" ]`

join: `", ".join(["Institute of Science and Engineering", "Shimane University", ",Matsue 690-8504, ", "Japan" ])`

result: `'Institute of Science and Engineering, Shimane University, ,Matsue 690-8504, , Japan'`

File: hepcrawl/extractors/jats.py (hunk 1 above)
Comment (ErnestaP) [reply hepcrawl/extractors/jats.py]: I removed this part completely

**Pull request merged**